libs/datamodel/core/tests/config/sources.rs

```diff
@@ -3,9 +3,12 @@ use datamodel::{ast::Span, error::DatamodelError};
 use pretty_assertions::assert_eq;
 use serial_test::serial;
 
-const DATAMODEL: &str = r#"
+#[test]
+#[serial]
```

Comment:
Ohh, I didn't know about [serial-test](https://github.com/palfrey/serial_test)! This is really neat!